apps/pom.xml

```diff
@@ -971,9 +971,9 @@ checkJavaFormat
 										</pluginExecution>
 										<pluginExecution>
 											<pluginExecutionFilter>
-												<groupId>com.coveo</groupId>
+                                                                                               <groupId>com.spotify.fmt</groupId>
 												<artifactId>fmt-maven-plugin</artifactId>
```

Comment:
but if you view the file or bring it down, it lines up perfectly.

Reply:
> but if you view the file or bring it down, it lines up perfectly.

Maybe, but that depends on the settings of your local text editor/IDE. It appears that `apps/pom.xml` uses tabs rather than spaces for indentation. Can you update this line to be consistent with the rest of the document?

Reply:
yes sir

Reply:
should be tabs now